public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    },
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
```

Comment:
This seems like it's from the sample app, I think we'd probably like to modify some of this before it goes into master. The names (name, shortname) should correspond to our project name. Do we want/need to have all these logos? I imagine the react logos are for their sample app, not sure if we have to keep those.

Reply:
I left it in because I was just committing the unedited changes from running the react command, but I think you're right. I got rid of the things we for sure won't need for our project.